src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -51,14 +51,14 @@ internal IArgumentOperation CreateArgumentOperation(ArgumentKind kind, IParamete
                 isImplicit: expression.WasCompilerGenerated || argument == null);
         }
 
-        private IVariableDeclarationOperation CreateVariableDeclarationInternal(BoundLocalDeclaration boundLocalDeclaration, SyntaxNode syntax)
+        private IVariableDeclaratorOperation CreateVariableDeclarationInternal(BoundLocalDeclaration boundLocalDeclaration, SyntaxNode syntax)
```

Comment:
method name mismatch?

Reply:
Fixed.

---
In reply to: [148087839](https://github.com/dotnet/roslyn/pull/22842#discussion_r148087839) [](ancestors = 148087839)